config/initializers/assets.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.config.assets.precompile += [
+  "application_split2.css",
```

Comment:
Can we keep a comment mentioning `css_splitter` here? Especially as it's potentially going away soon, greppability is good.